README.md

```diff
@@ -396,7 +389,4 @@ Credentials.
 
 [gcs]: https://cloud.google.com/storage
 [wif]: https://cloud.google.com/iam/docs/workload-identity-federation
-[sa]: https://cloud.google.com/iam/docs/creating-managing-service-accounts
-[gh-runners]: https://help.github.com/en/actions/hosting-your-own-runners/about-self-hosted-runners
-[gh-secret]: https://help.github.com/en/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets
-[setup-gcloud]: ../setup-gcloud
+[sa]: https://cloud.google.com/iam/docs/creating-managing-service-accounts
```

Comment:
Can you do a once over to make sure all references to v0 are updated to v1, including for this action. This is a chicken vs egg problem but we can update in one shot prior to releasing.